libs/portal-integration-angular/src/lib/core/components/diagram/diagram.component.ts

```diff
@@ -2,31 +2,59 @@ import { Component, EventEmitter, Input, OnChanges, OnInit, Output } from '@angu
 import { TranslateService } from '@ngx-translate/core'
 import { ChartData, ChartOptions } from 'chart.js'
 import * as d3 from 'd3-scale-chromatic'
-import { ColorUtils } from '../../utils/colorutils'
+import { PrimeIcons } from 'primeng/api'
 import { DiagramData } from '../../../model/diagram-data'
 import { DiagramType } from '../../../model/diagram-type'
+import { ColorUtils } from '../../utils/colorutils'
+
+export interface DiagramLayouts {
```

Comment:
Why is this exported?

Reply:
I use this to strongly type the mock data created in `diagram.component.spec.ts` If necessary I can however also remove the references in my tests and stop exporting the interface.